content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,116 @@
+workloads.
+
+Ambient mode works without any modification required from your existing Kubernetes deployments. Users can label a namespace to
+add all of its workloads to Istio’s ambient mode, or opt-out of certain deployments as needed. By utilizing ambient mode, users
+bypass some of the previously restrictive elements of the sidecar model and instead can now expect server send-first protocols
+to work, see most of the reserved ports are now removed, and the ability for containers to bypass the sidecar – either
+maliciously or not – is greatly reduced.
+
+As part of the 1.22 release of Istio, the announcement of ambient mode in Beta also includes how this latest release brings
+both Layer 4 and Layer 7 mesh features to production readiness without sidecars.
+
+The lightweight shared L4 node proxy is called the zero-trust tunnel, or ztunnel. Ztunnel drastically reduces the overhead of
+running a mesh by removing the need to potentially over provision memory and CPU within a cluster to handle expected loads. In
+some use cases, the savings can exceed 90% or more, while still providing zero-trust security using mutual TLS with
+cryptographic identity, simple L4 authorization policies, and telemetry.
+
+The L7 proxies are called waypoints. Waypoints process L7 functions such as traffic routing, rich authorization policy
+enforcement, and enterprise-grade resilience. Waypoints run outside of your application deployments and can scale independently
+based on your needs, which could be for the entire namespace or for multiple services within the namespace. Compared with
+sidecars, you don’t need 1 waypoint proxy per application pod, and you can scale your waypoint effectively based on its scope,
+thus saving CPU and memory significantly in many cases.
+
+The separation between the L4 secure overlay layer and L7 processing layer allows incremental adoption of the ambient mode data
+plane in contrast to the earlier binary “all in” injection of sidecars. Users can start with the secure overlay layer which
+offers mTLS with cryptographic identity, simple L4 authorization policy, and telemetry. Later on, complex L7 handling such as
+retries, traffic splitting, complex load balancing, and observability collection can be enabled on a case-by-case basis.
+
+## What is in the scope of the Beta?
+
+We recommend you explore the following Beta functions of ambient mode in production with appropriate caution, after validating
+them in test environments:
+
+- [Install](/docs/ambient/install/).
+- [Adding your workloads to the mesh](/docs/ambient/usage/add-workloads/) to gain mutual TLS with cryptographic identity, [L4 authorization policies](t/docs/ambient/usage/l4-policy/), and telemetry.
+- [Configure waypoint proxies](/docs/ambient/usage/waypoint/) to [use L7 functions](/docs/ambient/usage/l7-features/) such as traffic shifting, request routing, rich authorization policy enforcement.
```

Comment:
```suggestion
- [Configure waypoints](/docs/ambient/usage/waypoint/) to [use L7 functions](/docs/ambient/usage/l7-features/) such as traffic shifting, request routing, rich authorization policy enforcement.
```

Nit for consistency, these are called waypoints throughout the blog but then `waypoint proxy` here. We should either call them `waypoint proxy` or `waypoint`.